flow/transactions/send/send.go

```diff
@@ -82,20 +89,20 @@ func initConfig() {
 }
 
 func validateKeyPreReq(account *cli.Account) {
-	if account.KeyType == cli.KeyTypeHex {
+	if account.DefaultKey().Type() == config.KeyTypeHex {
 		// Always Valid
-		return
-	} else if account.KeyType == cli.KeyTypeKMS {
+		return // TODO: check difference between googleKMS and KMS
```

Comment:
In this case, Google KMS and KMS are the same thing. I think we should always include "Google" in the name, in case we choose to implement other key management services (KMS) in the future.